backend/src/models/User.js

```diff
@@ -0,0 +1,23 @@
+import { Sequelize, DataTypes } from 'sequelize';
+import { hashPassword } from '../lib/auth.js';
+import safeAwait from 'safe-await';
+
+module.exports = (sequelize, DataTypes) => {
```

Comment:
Because we're using ES6 imports, I would probably use `export default (seqelize, DataTypes) => ...`